arduino-ide-extension/src/electron-main/theia/electron-main-application.ts

```diff
@@ -339,6 +363,9 @@ export class ElectronMainApplication extends TheiaElectronMainApplication {
       options.webPreferences = {};
     }
     options.webPreferences.v8CacheOptions = 'bypassHeatCheck'; // TODO: verify this. VS Code use this V8 option.
+    options.useContentSize = true;
+    options.minWidth = 680;
+    options.minHeight = 565;
```

Comment:
Could you override `getDefaultTheiaWindowOptions` and do the customization there?

Reply:
Yes I could, but can I ask you why do you think it's better to set this option inside `getDefaultTheiaWindowOptions` rather than `getDefaultOptions`?

My rationale to put it inside `getDefaultOptions` is that Theia sets `minWidth` and `minHeight` there.

Reply:
You're absolutely right. Thanks for asking for clarification.